common/test/acceptance/pages/lms/teams.py

```diff
@@ -315,6 +322,13 @@ def team_user_membership_text(self):
         query = self.q(css='.page-content-secondary > .team-user-membership-status')
         return query.text[0] if query.present else ''
 
+    def wait_for_leave_team_button_presence(self):
+        """ Helper method to wait for Leave Team button to appear """
+        self.wait_for(
```

Comment:
Yeah, I don't have a strong preference either way, as long as we consistently do `wait_for_leave_team_button_presence` and `wait_for_join_team_button_presence`the same way.